integrationtest/src/test/java/org/hibernate/ogm/test/integration/infinispan/InfinispanModuleMemberRegistrationIT.java

```diff
@@ -41,6 +41,7 @@ private static PersistenceDescriptor persistenceXml() {
 				.name( "primary" )
 				.provider( "org.hibernate.ogm.jpa.HibernateOgmPersistence" )
 				.clazz( Member.class.getName() ).getOrCreateProperties()
+				.createProperty().name( "jboss.as.jpa.providerModule" ).value( "application" ).up()
```

Comment:
What's that one needed for? Specifically I can't seem to find any docs on the "application" value. I suppose your docs will clarify it :)